src/MacroTools/Mechanics/DemonGates/DemonGateBuff.cs

```diff
@@ -130,10 +131,11 @@ private void SpawnDemon()
     {
       for (var i = 0; i < _spawnCount; i++)
       {
-        var spawnedDemon = CreateUnit(Target.OwningPlayer(), _demonUnitTypeId, SpawnPoint.X, SpawnPoint.Y,
-          Target.GetFacing() + FacingOffset)
-          .IssueOrder("attack", RallyPoint);
+        var spawnedDemon = CreateUnit(Target.OwningPlayer(), _demonUnitTypeId, SpawnPoint.X, SpawnPoint.Y, Target.GetFacing() + FacingOffset)
+          .IssueOrder(851983, RallyPoint);
```

Comment:
Refer to my other comment on using `IssueOrder`

Reply:
Amended, as above